modules/statistics/php/stats_demographic.class.inc

```diff
@@ -55,38 +56,40 @@ class Stats_Demographic extends \NDB_Form
     /**
      * RenderStatsTable function
      *
-     * @param string $sectionHeader     the value of sectionHeader
-     * @param string $tableHeader       the value of tableHeader
-     * @param string $subcats           the value of subcats
-     * @param string $visits            the value of visits
-     * @param string $dropdown_name     the value of dropdown_name
-     * @param string $dropdown_opt      the value of dropdown_opt
-     * @param string $dropdown_selected the value of dropdown_selected
-     * @param string $centres           the value of centres
-     * @param string $data              the value of data
-     * @param string $Subsection        the value of Subsection=''
-     * @param string $disclamer         the value of disclamer=''
-     * @param string $projectID         the value of projectID is null
+     * @param string $sectionHeader    the value of sectionHeader
+     * @param string $tableHeader      the value of tableHeader
+     * @param string $subcats          the value of subcats
+     * @param string $visits           the value of visits
+     * @param string $dropdownName     the value of dropdownName
+     * @param string $dropdownOpt      the value of dropdownOpt
+     * @param string $dropdownSelected the value of dropdownSelected
+     * @param string $centers          the value of centers
+     * @param string $data             the value of data
+     * @param string $subsection       the value of subsection=''
+     * @param string $disclamer        the value of disclamer=''
+     * @param string $projectID        the value of projectID is null
      *
      * @return string
      */
     function renderStatsTable($sectionHeader,
         $tableHeader,
         $subcats,
         $visits,
-        $dropdown_name,
-        $dropdown_opt,
-        $dropdown_selected,
-        $centres,
+        $dropdownName,
+        $dropdownOpt,
+        $dropdownSelected,
+        $centers,
         $data,
-        $Subsection="",
+        $subsection="",
```

Comment:
this is such a minor annoyance for me haha but could we stick with either single quotes or double quotes

Reply:
yes will do